pages/account/manage/profile.js

```diff
@@ -44,6 +46,16 @@ export async function getServerSideProps(context) {
     profile.name = session.user.name;
   }
 
+  //reroute to profile deactivated page if profile is disabled
+  if(profile && !profile.isEnabled) {
+    return {
+      redirect: {
+        destination: "/account/profile-deactivated",
+        permanent: false,
+      }
+    }
+  }
+
```

Comment:
I can't create a new profile since new user is not enabled by default. Additionally, there is a check for `if (profile && !profile.isEnabled)` in all account pages, so I can't setup a profile.

Reply:
Hey how can i test the user creation flow? Generally i connect my account locally by the help of access token generated from github.

Reply:
Hi, sorry for the long reply. I use Github OAUth keys for local development. so, i can create a profile. here's more info about it https://linkfree.io/docs/environments/environment-variables

Reply:
we are specifying `isEnabled` flag default value as `true` in `Profile` schema, so ideally it should inject default value for a new user. not sure why is this going wrong. i am kind of stuck at this issue, can you please guide me anyway possible ?

Reply:
@webAbhi come and chat to us in EddieHub Discord, we can help better there